easybuild/easyconfigs/l/LavaVu/LavaVu-1.9.9-foss-2024a-egl.eb

```diff
@@ -0,0 +1,76 @@
+# Thanks to Mikael Öhman, Alexandre Strube and Alex Domingo for helping with
+# the Python problem
+# Author: J. Saßmannshausen (Imperial College London / UK)
+# Use this version for any GUI installations you want to do
+
+easyblock = 'PythonBundle'
+
+name = 'LavaVu'
+version = '1.9.9'
+versionsuffix = '-egl'
+
+homepage = 'https://lavavu.github.io/Documentation/'
+description = """
+A lightweight scientific visualisation tool with a python interface for fast
+and flexible visual analysis.
+
+LavaVu provides access to OpenGL powered 3D rendering of point, line, surface
+and volume data with minimal setup and an interface focused on simple mapping
+of values to visual properties while abstracting all the intricacies of OpenGL.
+A scientific visualisation tool with a python interface for fast and flexible
+visual analysis.
+"""
+citing = """
+[1] Stegman, D.R., Moresi, L., Turnbull, R., Giordani, J., Sunter, P., Lo, A.
+and S. Quenette
+gLucifer: Next Generation Visualization Framework for High performance
+computational geodynamics, 2008, Visual Geosciences
+http://dx.doi.org/10.1007/s10069-008-0010-2
+[2] Ruijters, Daniel & ter Haar Romeny, Bart & Suetens, Paul. (2008).
+Efficient GPU-Based Texture Interpolation using Uniform B-Splines. J. Graphics Tools. 13. 61-69.
+http://dx.doi.org/10.1080/2151237X.2008.10129269
+
+Zenodo: https://doi.org/10.5281/zenodo.2585376
+"""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+builddependencies = [
+    ('SWIG', '4.2.1'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('X11', '20240607'),
+    ('FFmpeg', '7.0.2'),
+    ('libglvnd', '1.7.0'),
+    ('aiohttp', '3.10.10'),
+    ('jupyter-server-proxy', '4.4.0'),
+    ('matplotlib', '3.9.2'),
+    ('Mesa', '24.1.3'),
+]
+
+exts_list = [
+    ('numpy_quaternion', '2024.0.9', {
+        'modulename': 'quaternion',
+        'checksums': ['de0125055bb2fed019c1afe8fe5a9844739a46196a1ee8838fa6b6b2a149aaad'],
+    }),
+    ('lavavu', version, {
+        # Possible choices: EGL, OSMESA or X11
+        'prebuildopts': 'export LV_EGL=1 && ',
+        # The original symlink leads to problems, so we are doing this dance to get that fixed:
+        'postinstallcmds': [
+            'rm -f %(installdir)s/lib/python%(pyshortver)s/site-packages/lavavu/osmesa/LavaVuPython.py && '
+            'cp -f %(installdir)s/lib/python%(pyshortver)s/site-packages/lavavu/LavaVuPython.py '
+            '%(installdir)s/lib/python%(pyshortver)s/site-packages/lavavu/osmesa/ &&'
+            'cd  %(installdir)s/lib/python%(pyshortver)s/site-packages/lavavu/osmesa/ && '
+            'sed -i "9,11d" LavaVuPython.py && sed -i "s/    import _LavaVuPython/import _LavaVuPython/" '
```

Comment:
I am trying to think if there would be a better way to future proof this.

- Best case scenario they never move these lines
- Middle case scenario they move the lines, but we fail the sanity check 
- Worst case scenario they move the lines in a way that the sanity check still passes but we introduce an error in they way the code works

Would it be possible to apply this as a patch?
That way we enforce the match of the original lines and if it does not the build fails at the start (instead at the end in the middle-case scenario or not at all in the worst-case)